modules/ebs_snapshot_explorer/main.py

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env python3
+"""Module for ebs_snapshot_explorer"""
+from copy import deepcopy
+import json
+from pathlib import Path
+
+from botocore.exceptions import ClientError
+
+from . import parser
+from . import module_info
+
+
+def input_helper(input):
+    """Helper function that loops until a successful response is given"""
+    prompt = '    Load next set of volumes? (y/n) '
+    while True:
+        response = input(prompt)
+        if response.lower() == 'y':
+            return True
+        elif response.lower() == 'n':
+            return False
+
+def load_volumes(client, print, input, instance_id, volume_ids):
+    """Loads volumes on an instance.
+
+    Args:
+        client (boto3.client): client to interact with AWS
+        print (func): Overwritten built-in print function
+        input (func): Overwritten built-in input function
+        instance_id (str): instance_id to attach volumes to
+        volume_ids (list): list of volume_ids to attach to the instance.
+    Returns:
+        bool: True if all volumes were successfully attached.
+    """
+
+    # load volume set
+    set_index = 0
+    set_count = 10
+
+    while set_index < len(volume_ids):
+        current_volume_set = volume_ids[set_index:set_index + set_count]
+        waiter = client.get_waiter('volume_available')
+        waiter.wait(VolumeIds=current_volume_set)
+        attached = modify_volume_set(
+            client, print, 'attach_volume', instance_id, current_volume_set)
+        if not attached:
+            print(' Volume attachment failed')
+            print(' Exiting...')
+            return False
+        running = input_helper(input)
+        detached = modify_volume_set(
+            client, print, 'detach_volume', instance_id, current_volume_set)
+        if not detached:
+            print(' Volume detachment failed')
+            print(' Exiting...')
+            return False
+        waiter.wait(VolumeIds=current_volume_set)
+        set_index += set_count
+        if not running:
+            break
+    cleanup(client)
+    return True
+
+
+def modify_volume_set(client, print, func, instance_id, volume_id_set):
```

Comment:
volume_id_set here, volume_ids on line 74

A set is a python type, so try to avoid having "set" in the name if it is a list